tests/immu/test_session.py

```diff
@@ -0,0 +1,194 @@
+            assert what == [{"id": 1, "tester": '123'}, {"id": 2, "tester": '321'}]
+            commit = newTx.rollback()
+
+            newTx = session.newTx()
+            what = newTx.sqlQuery(f"SELECT * FROM {table}", dict(), columnNameMode=constants.COLUMN_NAME_MODE_FIELD)
+            assert what == [{"id": 1, "tester": '123'}]
+            commit = newTx.commit()
+
+    def test_unmanaged_session(self, wrappedClient: ImmuTestClient):
+        if(not wrappedClient.serverHigherOrEqualsToVersion("1.2.0")):
+            pytest.skip("Version too low")
+        currentTxInterface = wrappedClient.openSession("immudb", "immudb", b"defaultdb")
+        wrappedClient.currentTx = currentTxInterface
+        key = wrappedClient.generateKeyName().encode("utf-8")
+        a = wrappedClient.get(key)
+        assert a == None
+        a = wrappedClient.set(key, b'1')
+        a = wrappedClient.get(key)
+        assert a.value == b'1'
+        a = wrappedClient.get(key)
+        assert a.value == b'1'
+        interface = wrappedClient.newTx()
+        table = wrappedClient.createTestTable("id INTEGER AUTO_INCREMENT", "tester VARCHAR[10]", "PRIMARY KEY id")
+        wrappedClient.insertToTable(table, ["tester"], ["@blabla"], {"blabla": "3"})
+        wrappedClient.insertToTable(table, ["tester"], ["@blabla"], {"blabla": "4"})
+        wrappedClient.insertToTable(table, ["tester"], ["@blabla"], {"blabla": "5"})
+        interface.commit()
+        wrappedClient.closeSession()
+        currentTxInterface = wrappedClient.openSession("immudb", "immudb", b"defaultdb")
+        wrappedClient.currentTx = currentTxInterface
+        interface = wrappedClient.newTx()
+
+        what = wrappedClient.simpleSelect(table, ["tester"], dict())
+        concatenated = [item[0] for item in what]
+        assert concatenated == ["3", "4", "5"]
+
+        interface.commit()
+        interface = wrappedClient.newTx()
+        wrappedClient.insertToTable(table, ["tester"], ["@blabla"], {"blabla": "6"})
+        wrappedClient.insertToTable(table, ["tester"], ["@blabla"], {"blabla": "7"})
+        wrappedClient.insertToTable(table, ["tester"], ["@blabla"], {"blabla": "8"})
+        interface.rollback()
+        wrappedClient.closeSession()
+        wrappedClient.currentTx = wrappedClient.openSession("immudb", "immudb", b"defaultdb")
+        interface = wrappedClient.newTx()
+        what = wrappedClient.simpleSelect(table, ["tester"], dict())
+        concatenated = [item[0] for item in what]
+        assert concatenated == ["3", "4", "5"]
+
+        with wrappedClient.openManagedSession("immudb", "immudb", b"defaultdb") as session:
+            key = wrappedClient.generateKeyName().encode("utf-8")
+            a = wrappedClient.get(key)
+            assert a == None
+            a = wrappedClient.set(key, b'1')
+            a = wrappedClient.get(key)
+            assert a.value == b'1'
```

Comment:
Missing newline

Reply:
Done in https://github.com/codenotary/immudb-py/pull/46/commits/b53e7a2d899e34be511b255bc4fe0d6199864c61